packages/zoltan2/core/src/algorithms/zoltan/Zoltan2_AlgZoltan.hpp

```diff
@@ -596,6 +630,118 @@ class AlgZoltan : public Algorithm<Adapter>
     setCallbacksGeom(&(*adapter));
   }
 
+
+  // JAE: Not implemented yet
```

Comment:
WHat is not implemented here?  Just the hierarchical part?  Maybe move comment below.